src/main/scala/tech/cryptonomic/conseil/tezos/michelson/dto/MichelsonExpression.scala

```diff
@@ -25,7 +25,7 @@ sealed trait MichelsonExpression extends MichelsonElement
 case class MichelsonType(prim: String, args: List[MichelsonExpression] = List.empty) extends MichelsonExpression
 
 /* Class representing an int constant */
-case class MichelsonIntConstant(int: Int) extends MichelsonExpression
+case class MichelsonIntConstant(int: Long) extends MichelsonExpression
```

Comment:
just a thought about naming, you changed `Int` to `Long`, but name of the case class stayed the same, maybe it is worth to change it to something that does not imply specific scala type like that, to something like `MichelsonNumericConstant`, `MichelsonIntegerConstant`? WDYT?

Reply:
The problem is that `int` is the name of a field which contains long value encodes as a string ;)
That's what it looks like in Michelson and I agree that it's confusing. I'll look at this again tomorrow with a fresh head.

Reply:
So in this case it makes more sense 👍